docs/enhancement-proposals/off-ep-0005.md

```diff
@@ -2,118 +2,129 @@
+continuing to use this ambiguity. Implementations may wish to execute this up-conversion automatically (see below).
 
-## Backward compatibility
+If backward-compatibility is provided as specified below, users of old force fields will not need to update their force field definitions.
 
-Version 0.4 of this tag is backwards-incompatible with older versions because the information
-content is different. It is ambiguous how to map both `method_periodic` and `method_nonperiodic`
-back down to a single `method` and it is not recommended to try doing so.
+## Backward compatibility
 
-Implementations may wish to add up-converters from old versions. A common up-converter could convert the
-following tag header
+To minimize pain in implementations, we can allow implementations continue to accept `method="PME"` as a synonym for `periodic_potential="Ewald3D-ConductingBoundary"` for a few releases,
+issuing a warning that the spec has changed and this behavior may be removed in a future release.
 
+Implementations may wish to add up-converters from old versions. An up-converter could convert the following tag header
 ```
 <Electrostatics version="0.3" method="PME" scale12="0.0" scale13="0.0" scale14="0.833333" scale15="1.0"/>
 ```
-
 to a less ambiguous header using version 0.4, which for this case could be
-
 ```
-<Electrostatics version="0.4" method_periodic="PME" method_nonperiodic="Coulomb" scale12="0.0" scale13="0.0" scale14="0.833333" scale15="1.0"/>
+<Electrostatics version="0.4" periodic_potential="Ewald3D-ConductingBoundary" nonperiodic_potential="Coulomb" exception_potential="Coulomb" scale12="0.0" scale13="0.0" scale14="0.833333" scale15="1.0"/>
 ```
 
-This is likely desirable for implementations such as the OpenFF Toolkit in which the information
-content of the version 0.4 snippet represents what is it currently does.
-
 ## Detailed description
 
-The `method` tag attribute is **removed** and replaced with `method_periodic` and `method_nonperiodic`.
-The default values of each are `"PME"` and `"Coulomb"`, respectively. The following section is added
-to justify and describe these new methods:
+[CODATA 2018](https://physics.nist.gov/cuu/Constants/index.html) physical constants are used.
+Future OFF-EPs may migrate the specification of which self-consistent physical constants are used to a higher-level attribute.
 
-```
-Some methods for computing electrostatics interactions are not valid for periodic systems, so
-separate methods must be specified for periodic (`method_periodic`) and non-periodic
-(`method_nonperiodic`) systems.
-```
+The `method` tag attribute is **removed** and replaced with `periodic_potential`, `nonperiodic_potential`, and `exception_potential`.
+
+The optional `cutoff` tag attribute is added to specify the cutoff used for `periodic_potential` if applicable, defaulting to `None`.
+No value of `cutoff` is allowed for Ewald methods---only finite-ranged methods.
```

Comment:
I'm updating the SMIRNOFF spec text and realized that I don't understand this - The existing `Electrostatics` tag already has a `cutoff` attribute, which defaults to `9.0 * unit.angstrom`. Is the proposal here to split the existing `cutoff` attribute into `cutoff_periodic` and `cutoff_nonperiodic`?

Reply:
I think (especially if I wrote this, I don't know) that line 88 should refer to "`cutoff_potential` if applicable ..."and not allowed/ignored for Ewald/Ewald-X methods that do not have a concept of a cutoff.

A cutoff _is_ applicable for the niche use case of #6. I think for other non-periodic potentials ("no cutoff" is the only one that comes to mind) it does not apply.